app/assets/stylesheets/civil_main.scss

```diff
@@ -0,0 +1,4 @@
+.search-field{
```

Comment:
[Ditto](https://github.com/ministryofjustice/laa-apply-for-legalaid-api/pull/33/files#r226589637)